source/_integrations/knocki.markdown

```diff
@@ -16,12 +16,31 @@ ha_platforms:
 ha_integration_type: device
 ---
 
-The **Knocki** {% term integration %} can receive events for the triggers set up from your [Knocki devices](https://knocki.com/).
+The **Knocki** {% term integration %} allows you to trigger your favorite automations simply by tapping custom patterns (such as triple taps) on ordinary surfaces.
 
-## Prerequisites
+## Requirements
 
-To use the Knocki integration, you need a Knocki account.
+Before you begin the setup process, you must have a [Knocki device](http://knocki.com).
 
-Once you set a trigger in the app, it will show up in Home Assistant to use.
+## Setup Instructions
+
+Start the setup process in the knocki mobile app (steps 1-7), and complete the process in Home Assistant (steps 8-13).
+
+1. Open the Knocki Mobile App. It’s a free download on [iOS](https://apps.apple.com/us/app/knocki/id1238395440) and [Android](https://play.google.com/store/apps/details?id=com.knocki.mobileapp).
+2. Using the app, log-in or create a new Knocki account.
+3. From the app’s home screen, select a Knocki or [add a Knocki](https://support.knocki.com/hc/en-us/articles/12769368448659-How-Do-Add-a-Knocki-to-My-Account).
+4. On the app’s device screen, select a gesture or [add a gesture](https://support.knocki.com/hc/en-us/articles/360013333634-Creating-a-Gesture).
+5. Next, select the [add a task](https://support.knocki.com/hc/en-us/articles/12920956118291-How-do-I-Add-Tasks-to-a-Knocki) option.
+6. Scroll down the task menu and select Home Assistant.
+7. Follow the in-app instructions to add the Home Assistant task.
+8. In Home Assistant, add the Knocki Integration by clicking the blue "Add Integration" button.
+9. You should see entities for each Knocki Task that you already created in the Knocki app, and the corresponding Knocki device IDs. If any entities are missing, please select "reload" from the device’s menu.
+10. Go to settings in Home Assistant, and select "+ CREATE AUTOMATION" and then choose the "Create new automation" option.  
+11. Select "+ADD TRIGGER."  
```

Comment:
```suggestion 
```

Reply:
@c0ffeeca7 I think all your changes make sense. Now wondering if this should be added in some place, I think it's important. What do you think? Thanks!
> 9. You should see entities for each Knocki Task that you already created in the Knocki app, and the corresponding Knocki device IDs. If any entities are missing, please select "reload" from the device’s menu.

Reply:
In an ideal world this would not be able to happen since the integration would update that. And I think that that logic is quite stable now right?

Reply:
@joostlek correct. Maybe just:
> You should see entities for each Knocki Task that you already created in the Knocki app, and the corresponding Knocki device IDs.